sonnet/src/nets/dnc/control.py

```diff
@@ -100,7 +100,7 @@ def deep_core(control_name,
     num_layers: Number of layers.
     skip_connections: Boolean that indicates whether to use skip connections.
       See documenation for sonnet.DeepRnn in
-      //learning/deepmind/tensorflow/sonnet/python/modules/basic_rnn.py for more
```

Comment:
Thank you for this, I think we can do even better if we use RST to x-link to the actual symbol. Then in our rendered docs (https://sonnet.dev) users can click through to DeepRNN. Concretely, I suggest changing this to:

```
See documentation for :class:`DeepRNN` for more information.
```

Reply:
Thank you for this review. Have made the change in the script.